src/Skeletor/Packages/Package.php

```diff
@@ -135,9 +136,46 @@ public function setFacade(string $facade)
         $this->facade = $facade;
     }
 
+    /**
+     * @return mixed
+     */
+    public function getEnvironmentVariables()
+    {
+        return $this->envVariables;
+    }
+
+    /**
+     * @param array $vars
+     */
+    public function setEnvironmentVariables(array $envVariables)
+    {
+        $this->envVariables = $envVariables;
+    }
+
+    /**
+     * @return boolean
+     */
+    public function hasEnvironmentVariables()
+    {
+        return count($this->envVariables) > 0 ? true : false;
```

Comment:
We do not need the ternary operator, count($this->envVariables) > 0 already returns the boolean we are after